README.md

```diff
@@ -1 +1,6 @@
-# PyWeek-35
\ No newline at end of file
+# PyWeek-35
+
+# Attribution
+
+## Art
+- [Star Icon by pixel perfect](https://www.flaticon.com/free-icon/star_1828884?term=star&page=1&position=1&origin=tag&related_id=1828884)
```

Comment:
If you press on "How to attribute?"

Flaticon provides us with

```html
<a href="https://www.flaticon.com/free-icons/star" title="star icons">Star icons created by Pixel perfect - Flaticon</a>
```

So our attribution should be 
```md
- [Star icons created by Pixel perfect - Flaticon](https://www.flaticon.com/free-icons/star)
```

I will fix it